daemon/config.go

```diff
@@ -18,6 +18,11 @@ import (
 )
 
 const (
+	defaultMaxDownloadConcurrency = 5
+	defaultMaxUploadConcurrency   = 3
```

Comment:
I noticed that this reverses the old defaults. The old code was:

```
	// maxDownloadConcurrency is the maximum number of downloads that		
 	// may take place at a time for each pull.		
 	maxDownloadConcurrency = 3		
 	// maxUploadConcurrency is the maximum number of uploads that		
 	// may take place at a time for each push.		
 	maxUploadConcurrency = 5
```

I was a little surprised to see that there are more simultaneous uploads allowed than downloads, but I think I remember why it's like this. Registries often have high latencies to a storage backend, such as S3. At the end of a layer upload, there are multiple steps that happen to commit that upload, and because of the latency, this can take several seconds. If a push is uploading many small layers, it's faster overall to upload more at the same time, so less time is wasted in between uploads waiting for the registry to do these commits.